src/Microsoft.ML.Data/Evaluators/Metrics/RegressionMetrics.cs

```diff
@@ -18,7 +18,7 @@ public sealed class RegressionMetrics
         /// y'i are the predicted labels for each instance.
         /// yi are the correct labels of each instance.
         /// </remarks>
-        public double L1 { get; }
+        public double AbsoluteLoss { get; }
```

Comment:
>AbsoluteLoss [](start = 22, length = 12)

This is the `MeanAbsoluteError` if you want to move away from L1. (It's not really a loss because we're not fitting to it.)